src/web/statics.rs

```diff
@@ -252,7 +252,7 @@ mod tests {
         wrapper(|env| {
             let web = env.frontend();
 
-            let files = &[("pure-min.css", "text/css")];
+            let files = &[("highlightjs/styles/dark.min.css", "text/css")];
```

Comment:
Maybe test both files instead of only highlight.js?

Reply:
`/-/static/pure-min.css` has been removed in the last commit since it was not used anywhere.